### DISCUSSION TITLE
Mirror plugin

### DISCUSSION THREAD

**Discussion by vinhnguyen116** [Kong/kubernetes-ingress-controller#4124] [Q&A] [answered]
Hi everyone, 
I have checked arround and would a confirmation that is there any mirror plugin to mirror traffic from an endpoint ?

Regards,
vin

**Comment by programmer04** [comment, marked as answer]
Hello @vinhnguyen116,

currently there is no mirror traffic plugin available in [Kong Plugin Hub](https://docs.konghq.com/hub/). You can check some community solutions
- https://github.com/tarepanda1024/kong-plugin-http-mirror
- https://github.com/nhp0712/kong-plugin-http-mirror

they aren't officially endorsed by Kong.

*If the above answers your question please `Mark as answer` to increasing visibility for others.*

**Reply by vinhnguyen116** [reply]
Thank you for confirmation and response.
